docker/.env.example

```
@@ -19,6 +19,18 @@ POSTGRES_DB=postgres
 POSTGRES_PORT=5432
 # default user is postgres
 
+############
+# Supavisor -- Database pooler
+############
+POOLER_PROXY_PORT_TRANSACTION=6543
+POOLER_SECRET_KEY_BASE=UpNVntn3cDxHJpq99YMc1T1AQgQpc8kfYTuRgBiYa15BLrx8etQoXz3gZv1/u2oq
+POOLER_VAULT_ENC_KEY=your-encryption-key-32-chars-min
+POOLER_POOL_MODE=transaction
+POOLER_DEFAULT_POOL_SIZE=20
+POOLER_MAX_CLIENT_CONN=100
+POOLER_TENANT_ID=your-tenant-id
```

```suggestion
POOLER_DEFAULT_POOL_SIZE=20
POOLER_MAX_CLIENT_CONN=100
```

We are probably reduce the exposed params to just these 2. The rest can be inlined like https://github.com/supabase/supabase/blob/e9d6c2e7560989ba4b2922ec1e5697c73932b1b0/docker/docker-compose.yml#L217